sklearn/metrics/regression.py

```diff
@@ -573,3 +574,37 @@ def r2_score(y_true, y_pred, sample_weight=None,
         avg_weights = multioutput
 
     return np.average(output_scores, weights=avg_weights)
+
+
+def max_error(y_true, y_pred):
+    """
+    The Max-Error metric is the worst case error
+    between the predicted value and the true value.
+
+    Parameters
+    ----------
+    y_true : array-like of shape = (n_samples)
+        Ground truth (correct) target values.
+
+    y_pred : array-like of shape = (n_samples)
+        Estimated target values.
+
+    Returns
+    -------
+    max_error : float
+        A positive floating point value (the best value is 0.0).
+
+    Examples
+    --------
+    >>> from sklearn.metrics import max_error
+    >>> y_true = [3.1, 2.4, 7.6, 1.9]
+    >>> y_pred = [4.1, 2.3, 7.4, 1.7]
+    >>> max_error(y_true, y_pred)
+    1.0
+    """
+    y_type, y_true, y_pred, _ = _check_reg_targets(y_true, y_pred,
+                                                   'uniform_average')
+    if y_type == 'continuous-multioutput':
+        raise ValueError("Multioutput not supported in max_error")
+    max_error = np.around(np.max(np.abs(y_true - y_pred)), decimals=3)
```

Comment:
It seems strange to use ``np.around`` here. Why not providing users with the exact value?

Reply:
If i don't use `np.around` , i see results like this 
```
In [1]: import numpy as np
In [2]: a = np.array([3.1, 2.4, 7.6, 1.9])
In [3]: b = np.array([4.1, 2.3, 7.4, 1.7])
In [4]: a - b
Out[4]: array([-1. ,  0.1,  0.2,  0.2])
In [5]: np.abs(a-b)
Out[5]: array([1. , 0.1, 0.2, 0.2])
In [6]: np.max(np.abs(a-b))
Out[6]: 0.9999999999999996
```
I think it's an inherent thing with `numpy`, do you think this minute error is okay

Reply:
This is a common problem about floats, but I think we need to provide users with the exact value. You can change an example to pass the doctest (in normal tests, we can use things like ``assert_almost_equal``, but in the doctest we can't).

Reply:
Thanks . I have updated this.